internal/integration/script_test.go

```diff
@@ -197,6 +248,16 @@ func (t *pgTest) cmdExist(ts *testscript.TestScript, neg bool, args []string) {
 	})
 }
 
+func (t *liteTest) cmdExist(ts *testscript.TestScript, neg bool, args []string) {
+	cmdExist(ts, neg, args, func(_, name string) (bool, error) {
+		var b bool
+		if err := t.db.QueryRow("SELECT COUNT(*) FROM sqlite_master WHERE `type`='table' AND `name` = ?", name).Scan(&b); err != nil {
```

Comment:
are you scanning a number into a bool ?
why not scan into a number and return
```go
// query result scanned into var n int
...
...
}
return n > 0 ,nil
```

Reply:
I guess, because in SQL booleans are numbers. `>0 -> true`.

Reply:
exactly.